pkg/cluster/internal/kubeadm/config.go

```diff
@@ -74,11 +76,14 @@ clusterName: "{{.ClusterName}}"
 bootstrapTokens:
 - token: "{{ .Token }}"
 {{ if .ControlPlaneEndpoint -}}
-controlPlaneEndpoint: {{ .ControlPlaneEndpoint }}
+controlPlaneEndpoint: "{{ .ControlPlaneEndpoint }}"
 {{- end }}
 # we use a well know port for making the API server discoverable inside docker network. 
 # from the host machine such port will be accessible via a random local port instead.
```

Comment:
this comment is about the port.

Reply:
don't understand the comment, I had to quote the ControlPlaneEndpoint because it failed to create the JSON from the template using IPv6 addresses

Reply:
i wanted to say that the comment `# we use a well know port for making...` is now above advertiseAddress, while it applies to the port bellow.

Reply:
I think that the comment is ok, it's above the `api` section not above the port field, to explain that the `apiendpoint` uses a well know port.